Source/WebCore/platform/graphics/texmap/TextureMapper.h

```diff
@@ -50,9 +50,9 @@ enum class TextureMapperFlags : uint16_t;
 class TextureMapper {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    enum WrapMode {
-        StretchWrap,
-        RepeatWrap
+    enum class WrapMode {
```

Comment:
Do you need `: uint8_t` here ?

Reply:
I don't think it's needed but it doesn't hurt either